Application Connector Documentation: one click integration (#3696)
kyma-project/kyma: docs/application-connector/docs/08-02-get-client-certificate.md

}
```

> > **NOTE:** The response contains URLs to Application Registry and Events Service API, however, it is not recommeded to use these. You should call metadata endpoint URL, which is provided in `infoUrl` property, to fetch correct URLs to Application Registry and Events API, and other configuration details.

Reviewer: ```suggestion
> **NOTE:** The response contains URLs to the Application Registry and the Events Service API, however, it is not recommended to use them. Call the `metadata` endpoint URL, provided in the `infoUrl` property, to fetch the correct URLs and other configuration details.
```

> > **NOTE:** The response contains URLs to the Application Registry and Events Service API, however, it is not recommeded to use these. You should call the `metadata` endpoint URL, which is provided in `infoUrl` property, to fetch correct URLs to the Application Registry and Events API, and other configuration details.

Reviewer: ```suggestion
> **NOTE:** The response contains URLs to the Application Registry and the Events Service API, however, it is not recommended to use them. You should call the `metadata` endpoint URL, which is provided in `infoUrl` property, to fetch correct URLs to the Application Registry and to the Events API, and other configuration details.
```

Author: Done

After you receive the certificate, decode it and use it in your application. Register the services of your external solution through the Application Registry.

> ## Call metadata endpoint

Reviewer: ```suggestion
## Call the `metadata` endpoint
```

Reviewer: Could you also apply this suggestion?

> ## Call the `metadata` endpoint

Reviewer: ```suggestion
## Call the metadata endpoint
```

Author: Done

> You must call the metadata endpoint with the generated certificate to get URLs to the following:

Reviewer: ```suggestion
Call the `metadata` endpoint with the generated certificate to get URLs to the following:
```

Reviewer: Could you also apply this suggestion?

> You must call the `metadata` endpoint with the generated certificate to get URLs to the following:

Reviewer: ```suggestion
Call the `metadata` endpoint with the generated certificate to get URLs to the following:
```

Author: Done

> - Application Registry API

Reviewer: ```suggestion
- the Application Registry API
```

Author: Done

> - Events Service API

Reviewer: ```suggestion
- the Events Service API
```

Author: Done

> - Certificate renewal endpoint

Reviewer: ```suggestion
- the `certificate renewal` endpoint
```

Author: Done

> - Certificate revocation endpoint

Reviewer: ```suggestion
- the `certificate revocation` endpoint
```

Author: Done

> Use the link you got in the second step to fetch the metadata information. Run:

Reviewer: Maybe providing the exact link would be a better idea?

Author: Done

> Use the link you got in the second step to fetch the metadata information. Run:

Reviewer: ```suggestion
Use the link provided in the second step to fetch the metadata information. Run:
```
Do you mean this link `https://connector-service.kyma.local/v1/applications/signingRequests/info?token=h31IwJiLNjnbqIwTPnzLuNmFYsCZeUtVbUvYL2hVNh6kOqFlW9zkHnzxYFCpCExBZ_voGzUo6IVS_ExlZd4muQ==`?

Author: Done

```
curl {CLUSTER_DOMAIN}/v1/applications/management/info --cert {CERT_FILE_NAME}.crt --key {KEY_FILE_NAME}.key
```

> 

Reviewer: What is it in the code blocks?

Author: Done.

```
{
  "clientIdentity": {

>     "application": "APP_NAME"

Author: Done

  },
  "urls": {
    "metadataUrl": "https://gateway.{CLUSTER_DOMAIN}/{APP_NAME}/v1/metadata/services",
    "eventsUrl": "https://gateway.{CLUSTER_DOMAIN}/{APP_NAME}/v1/events",
    "renewCertUrl": "https://gateway.{CLUSTER_DOMAIN}/v1/applications/certificates/renewals",

>     "revokeCertUrl": "https://gateway.{CLUSTER_DOMAIN}/v1/applications/certificates/revocations"

Author: Done
